### DISCUSSION TITLE
How to Import Playlist from YouTube?

### DISCUSSION THREAD

**Discussion by Outthink9** [fast4x/RiMusic#3122] [Q&A] [answered]
Hello everyone.

I have a question how to import playlists from YouTube to RiMusic.

The procedure I currently do is:

_Go to the playlist in YTMusic > Share with RiMusic_

BUT the list that opens in RiMusic corresponds to the VIDEO versions of the songs. If the song has no video, well, it only shows the audio version, obviously.

Am I doing it wrong?

I leave a [Michael Jackson playlist](https://music.youtube.com/playlist?list=RDCLAK5uy_miD17xJM-5x1_5HaIJj44QMdIa0jJAp0E&playnext=1&si=V4TZKqzlPD6NM_N1) as an example.

Thank you all in advance!

**Comment by aneesh1122** [comment, marked as answer]
All the official YouTube playlists contain only videos. RiMusic can't do anything about it

**Reply by Outthink9** [reply]
I see... Thanks for you reply!

**Comment by fast4x** [comment]
Even if the playlist contains only video, it can be imported and listened to in audio.

**Reply by Outthink9** [reply]
Yes but there are some videos with a lot of monologues/dialogues and low quality audio. 

Nothing that can't be fixed manually.

Thanks for your work (and response)!!
